src/Jobs/DeleteTenantStorage.php

```
@@ -0,0 +1,47 @@
+        if (config('tenancy.filesystem.suffix_storage_path') === false) {
+            return;
+        }
+
+        $centralPath = tenancy()->central(fn () => storage_path());
+        $path = tenancy()->run($this->tenant, fn () => storage_path());
+
+        // Skip storage deletion if tenant's storage path is the same as central storage path
+        $tenantPathIsCentral = realpath($path) === realpath($centralPath);
+
+        if (is_dir($path) && ! $tenantPathIsCentral) {
+            File::deleteDirectory($path);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Facades\File' in method 'handle'. (undefined)

(StaticAccess)